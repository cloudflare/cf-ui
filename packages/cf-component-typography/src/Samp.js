import React from 'react';

import PropTypes from 'prop-types';

class Samp extends React.Component {
  render() {
    return (
      <samp className="cf-samp">
        {this.props.children}
      </samp>
    );
  }
}

Samp.propTypes = {
  children: PropTypes.node
};

export default Samp;
