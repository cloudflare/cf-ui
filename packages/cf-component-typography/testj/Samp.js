import React from 'react';
import renderer from 'react-test-renderer';
import Samp from '../src/Samp';

describe('Samp', () => {
  test('should render', () => {
    const component = renderer.create(<Samp>Samp</Samp>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
