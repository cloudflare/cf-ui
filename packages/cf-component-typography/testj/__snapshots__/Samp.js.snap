// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Samp should render 1`] = `
<samp
  className="cf-samp"
>
  Samp
</samp>
`;
